dashboard/app/views/levels/editors/_applab.html.haml

```diff
@@ -85,6 +85,12 @@
   :javascript
     levelbuilder.initializeCodeMirror('level_data_tables', 'javascript');
 
+.field
+  = f.label :data_library_tables,'Dataset Libraries'
+  Comma-separated list of dataset libraries you would like to pre-load in the level.
```

Comment:
If we wanted to make a dropdown of the dataset options so that levelbuilders don't have to type in the exact names they want, how much work would that be?
I don't think we need to do it right now, but it would probably be nice to have at some point